src/instance.rs

```diff
@@ -356,6 +356,9 @@ impl<T> Py<T> {
     }
 
     /// Deprecated alias for [`from_owned_ptr`](#method.from_owned_ptr).
+    ///
+    /// # Safety
+    /// `ptr` must be a pointer to a Python object of type T.
```

Comment:
Also, it should be noted that `ptr` must be owned. We can remove this function in the near future, though.

Reply:
Updated below (and above for `from_owned_ptr`)!